### PR TITLE
fix some typos in getting started page

### PR DESCRIPTION
**Pull request by Ayush7614**
This PR Contains

* Fixed some typos in Getting Started Page

Screenshot from 2022-07-04 14-35-02

### CODE REVIEW

**Comment by Ayush7614** [comment]
cc @GMishx

File: src/pages/Help/Overview/index.jsx [hunk 1]
```diff
@@ -68,7 +68,7 @@ const Overview = () => {
           </div>
           <div className="col-md-6 p-md-0">
             <h2 className="text-primary-color font-size-sub-heading">
-              FOSSology's Graphical User Interface
+              FOSSology Graphical User Interface
```
Comment (GMishx) [inline src/pages/Help/Overview/index.jsx:71]: Can you please explain the change? Existing sentence looks correct to me.

Comment (Ayush7614) [reply src/pages/Help/Overview/index.jsx:71]: Its Fossology`s and the correct sentence is Fossology

**Comment by shaheemazmalmmd** [comment]
Closing the pull request as there is no response from author.

**Pull request closed without merging**